### ISSUE TITLE
Error Unauthorized This action may only be performed with an elevated session.

### ISSUE THREAD

**Issue by LoriL2** [sjelfull/Craft-UserCreator#1]
Craft CMS 2.6.2944
UserCreator 1.0.2

When I tried to create a user using the plugin I received an error "Unauthorized This action may only be performed with an elevated session."

Thanks.
Lori

**Comment by sjelfull** [comment]
Thanks for reporting this. I had several reports on this today, and i'm currently investigating this.

**Comment by sjelfull** [comment]
This should be fixed in [release 1.0.3](https://github.com/sjelfull/Craft-UserCreator/releases/tag/1.0.3).

**Comment by LoriL2** [comment]
Your release 1.0.3 fixed the elevated session error, thank you.  However, I came across another issue when I created a user.  I added one row and left the "username" column empty thinking that because I have my system set to use the email as username I could.  Leaving the "username" column empty gave me the following error.

`CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'userId' cannot be null. The SQL statement executed was: INSERT INTO`craft_usergroups_users`(`groupId`,`userId`,`dateCreated`,`dateUpdated`,`uid`) VALUES (:row0_col0, NULL, :row0_col2, :row0_col3, :row0_col4). Bound with :row0_col0='3', :row0_col2='2016-09-28 15:52:00', :row0_col3='2016-09-28 15:52:00', :row0_col4='7b46a5fd-249f-4a87-ac87-f48cf2702c60'`

I have more to the error saved in a pdf if it would help you track down the issue.  When I tried again with the username column filled out it worked perfectly.
